tests/unit/TextUI/Output/Default/DefaultPrinterTest.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\TextUI\Output\Default;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Medium;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\TextUI\InvalidSocketException;
+use PHPUnit\TextUI\Output\DefaultPrinter;
+
+#[CoversClass(DefaultPrinter::class)]
+#[Medium]
+final class DefaultPrinterTest extends TestCase
+{
+    public static function providePrinter(): array
+    {
+        return [
+            [DefaultPrinter::standardOutput()],
+            [DefaultPrinter::standardError()],
+            [DefaultPrinter::from('socket://hostname:port')],
```

Comment:
this test failed before the fix in https://github.com/sebastianbergmann/phpunit/pull/5777#discussion_r1544726955

Reply:
to be more clear: I think the default printer would have created a fatal runtime error when used with a socket url before the recent readonly property fix.

I wonder whether its a feature used in the wild, as such a fatal runtime error would have been reported by someone - I guess..?

Reply:
I have never seen it used in the wild. I implemented it 10+ years ago and used it myself once or twice.